config/config.go

```diff
@@ -154,6 +155,26 @@ func LoadConfig(fname string) (*Config, error) {
 
 	log.Root().SetHandler(log.LvlFilterHandler(logLvl, log.Root().GetHandler()))
 
+	log.Error("this is the page size", "ps", cfg.PageSize)
+	if cfg.PageSize == 0 {
+		cfg.PageSize = 4096
+	}
+	if cfg.PageSize > 65536 {
+		log.Warn("config: pragma page size miust be less than or equal to 65536, setting to largest possible value: 65536")
+		cfg.PageSize = 65536
+	}
+	if cfg.PageSize > 0 && (cfg.PageSize&(cfg.PageSize-1)) != 0 {
```

Comment:
I'm inclined to just log a warning then let sqlite do what it's going to do with non powers of two, rather than try to find some specific value that they might have meant.